doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -49,6 +49,8 @@ Build and configuration system
 * Removed support for the deprecated multi-image builds (parent-child images) functionality.
   All |NCS| projects must now use :ref:`sysbuild`.
   See :ref:`child_parent_to_sysbuild_migration` for an overview of differences with parent-child image and how to migrate.
+* Updated the default runner for the ``west flash`` command to `nRF Util`_ instead of ``nrfjprog``, which is part of the archived `nRF Command Line Tools`_.
```

Comment:
```suggestion
* Updated the default runner for the ``west flash`` command to `nRF Util`_ instead of ``nrfjprog`` that is part of the archived `nRF Command Line Tools`_.
```